resources/js/lang/en.js

```diff
@@ -33,12 +33,10 @@ export default {
                 2: "The Bandit Commander's proclamations of a \"Gloom\" during your battle do not sit well with you. Sifting through the carnage left by the battle, you found a cache of scrolls. The writing upon them was in some unknown archaic language. Among the scrolls, you found a map of the northern countryside. A bend in the Still River was clearly marked as a point of interest. Contemplating whether you want to find out more about this so-called \"Gloom,\" you decided that the spot could be a point of interest for you, as well.{br}The map lead to a crypt full of demons and cultists. Clearly, the bandits are associated with some sort of cult. Your arrival disrupted some sort of ritual. Among the scripts found in the ritual chamber are notes on a couple of other places of power in the area. One is marked as being overrun by vicious undead. It looks as though you have an opportunity to get in the cult's good graces by helping to clear out a threat. This may help you learn more about the \"Gloom\".{br}After the terrible shrieks and moans of the undead ceased, a grinning hooded figure appeared before you. His demeanor was deeply unsettling. He asked another favor of you in exchange for forgiveness for killing his brethren. The cultists want the necromancer, Jekserah, dead. If you bring her head to their headquarters, you may find out more about the \"Gloom\". The figure gave you details on where to find evidence of Jekserah's plans.",
-                3: "The Bandit Commander's proclamations of a \"Gloom\" during your battle do not sit well with you. Sifting through the carnage left by the battle, you found a cache of scrolls. The writing upon them was in some unknown archaic language. Among the scrolls, you found a map of the northern countryside. A bend in the Still River was clearly marked as a point of interest. Contemplating whether you want to find out more about this so-called \"Gloom,\" you decided that the spot could be a point of interest for you, as well.{br}You have slain Jekserah. As she collapsed, the life fleeing her eyes, she whispered at you “You have no idea what’s coming...”. Perhaps your old boss's head may prove useful in future dealings with the bandits.",
```

Comment:
I removed these two sections because they only provided spoilers when Jekserah is killed and you didn't meet de hooded stranger yet. No reason to add this sentence before you're asked to kill her.

Reply:
Ok, I understand where you are coming from, but the lines "You have no idea what's coming" are sorta significant and hint towards a new Scenario (#28). I feel that it is kinda important that to indicate that there is more to come.

Reply:
I do agree that this quote is significant. But it still seems like a spoiler to me to put this quote together with this quest. You actually don't know what her quote applies to. So to show it on scenario 4 seems like a spoiler, it's at best a guess at that point that Jekserah was referring to this quest/scenario.

Reply:
I understand where you are coming from:
Until you do scenario 6 there is no price on Jekserah's head from the Bandits, so why would turning against Jekserah have any significance for them.

But on the other hand, the entire campaign started with Jekserah hiring you to steal back the scrolls about the Gloom from them. So, before killing her you had two people/groups connected to the Gloom: The bandits and Jekserah. With her out of the picture, the bandits are the only lead on the Gloom. I think this connection can be made in the quest.

I'll try to think of a way to include Jekserah in the storyline without being too spoilery. The whole part about her head needs to be removed anyway, because you could do scenario 20 before doing 6, in which case her head has already been given to the Captain of the Guard, and can't be given to the bandits anymore.

Reply:
I never made that connection that Jekserah was an opposition to the bandits as a group. Not until her head was requested. It is quite clear indeed when I read it again but when we just started playing gloomhaven it wasn't clear to us that the bandits where a collective instead of generic bandits. Seems legit to restore some of the texts that I've removed.

Reply:
I've reverted the commit that deleted these two stages and I've created a new issue to improve these texts.